fix(win-installer): 安装流程缺失服务条款/隐私协议展示，合规性不足

## 问题描述
Sudowork Windows 系统安装过程中未展示服务条款 / 隐私协议链接，用户知情权缺失，合规性不足。

## 预期行为
安装流程应展示服务条款和隐私协议链接，供用户阅读并确认。

## 环境
- Windows 系统

🤖

@claude windows 编译 Command line defined: "BUILD_UNINSTALLER"
Command line defined: "UNINSTALLER_OUT_FILE=D:\a\sudowork\sudowork\out\Sudowork-0.1.3-win-x64.__uninstaller.exe"
Processing config: D:\a\_temp\.cache\electron-builder\nsis\nsis-3.0.4.1-nsis-3.0.4.1\nsisconf.nsh
Processing script file: "<stdin>" (UTF8)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Processed 1 file, 11 command line commands, writing output (x86-unicode):
warning 6001: Variable "tosPage.Dialog" not referenced or never set, wasting memory!
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Note: you may have one or two (large) stale temporary file(s) left in your temporary directory (Generally this only happens on Windows 9x).
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Error output:
Error: warning treated as error
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        at ChildProcess.<anonymous> (D:\a\sudowork\sudowork\node_modules\builder-util\src\util.ts:272:14)
    at Object.onceWrapper (node:events:634:26)
    at ChildProcess.emit (node:events:519:28)
    at ChildProcess.cp.emit (D:\a\sudowork\sudowork\node_modules\cross-spawn\lib\enoent.js:34:29)
    at maybeClose (node:internal/child_process:1101:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)
❌ Build failed: Command failed: npx electron-builder --win --x64 --publish=never
Windows build completed successfully
Error: Process completed with exit code 1.

--- a/scripts/installer.nsh
+++ b/scripts/installer.nsh
@@ -17,9 +17,15 @@
   Var /GLOBAL deleteNexusData
 
   ; Variables for the Terms of Service / Privacy Policy agreement page
+  ; Guard with !ifndef BUILD_UNINSTALLER because these variables are only used
+  ; in the installer's custom ToS page (also guarded by !ifndef BUILD_UNINSTALLER).
+  ; Without this guard, the uninstaller pass triggers NSIS warning 6001
+  ; ("Variable not referenced or never set") which is treated as a build error.
+  !ifndef BUILD_UNINSTALLER
   Var /GLOBAL tosPage.Dialog
   Var /GLOBAL tosPage.Checkbox
   Var /GLOBAL tosPage.TextBox
+  !endif
 
   ; Override MUI2 uninstaller page strings for Simplified Chinese (LANG_SIMPCHINESE)
   ; Suppress warning 6030 (LangString set multiple times) since we intentionally

